WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -828,7 +828,10 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
         val context = activity ?: return
         val options = UCrop.Options()
         options.setShowCropGrid(false)
-        options.setStatusBarColor(context.getColorFromAttribute(android.R.attr.statusBarColor))
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.VANILLA_ICE_CREAM) {
+            @Suppress("DEPRECATION")
+            options.setStatusBarColor(context.getColorFromAttribute(android.R.attr.statusBarColor))
```

Comment:
Question (❓): So, for API 35 and beyond we don't set the status bar color, nor care for that? Can you share a bit more on that. 🙏

Reply:
Good catch, I meant to revisit that! I changed this in [42e2169](https://github.com/wordpress-mobile/WordPress-Android/pull/21599/commits/42e21690072014ab019a80082fabf4682fa0e1c0) so the status bar color is `colorSurface` for SDK 35+ and `colorOnSurface` for older versions.  I'm not sure if this is even used any more, though - I don't see uCrop changing the status bar color regardless of SDK version.

Reply:
Awesome, thanks @nbradbury ! ❤️

> I'm not sure if this is even used any more, though - I don't see uCrop changing the status bar color regardless of SDK version.

🤷